### DISCUSSION TITLE
What does `spy` stand for in `spy.yaml` model files names?

### DISCUSSION THREAD

**Discussion by proshin-roman** [serverpod/serverpod#1961] [Q&A] [answered]
Just out of curiosity: what does exactly `spy` mean in the model files names? I couldn't find anything in the documentation, neither here on GitHub. Is that an abbreviation of something (ServerPodYaml???)?

**Comment by vlidholt** [comment, marked as answer]
Yes, you are right. ServerPodYaml is what it is. If you use the Serverpod extension for VS Code, it will automatically recognize and format any file ending with `.yaml` that is located in the `models` directory. But for files to be recognized elsewhere they need to be either `.spy.yaml` or `.spy`

**Reply by proshin-roman** [reply]
Thank you, @vlidholt! So it's not necessary to have that `spy` suffix as long as the files are in the `models` directly - because I noticed that it all works in both ways for me 👍
